action.yml

```diff
@@ -38,3 +38,14 @@ inputs:
       as md5 hashes. The first matching revision that has a build will be the
       build that this build is pushed onto.
     required: false
+outputs:
+  outcomes:
+    description: >-
+      JSON-stringified object of build outcomes. Keys are languages, and values
+      contain the `commit` result of the build for that language.
+  parent_outcomes:
+    description: >-
+      JSON-stringified array, where each entry is an object of parent revision
+      build outcomes. Entries may be null if no matching parent build was found.
+      Keys are languages, and values contain the `commit` result of the build
+      for that language.
```

Comment:
idk if this is particularly actionable but I find these comments pretty confusing and I figure it would be confusing to the user as well, maybe just a short example, e.g.

``````
```
{
  "typescript": {
    "commit": {
      "sha": "1234",
    	...the commit object
   	}
  },
  "python": {
    "commit": {
      "sha": "1234",
    	...the commit object
   	}
  },
}
```
``````

would go a longish way?

Reply:
eh, i think as long as the api it's based on is a v0 then we can keep this a v0